### ISSUE TITLE
TextField/Dropdown border width on focus inconsistent with Picker components

### ISSUE THREAD

**Issue by onlyann** [OfficeDev/office-ui-fabric-react#11147]
### Environment Information

- **Package version(s)**: 7.59.3

The issue has been introduced in #10885

### Please provide a reproduction of the bug in a codepen:

See the picker example page https://developer.microsoft.com/en-us/fabric#/controls/web/peoplepicker

#### Actual behavior:

Notice how the border width for the picker is 1px wide while TextField and Dropdown components have a 2px border width when the field has focus.
![image](https://user-images.githubusercontent.com/1916218/68570602-c8ebbd00-04ac-11ea-9cfa-26be83890fe8.png)

![image](https://user-images.githubusercontent.com/1916218/68570627-d6a14280-04ac-11ea-80a2-25684b7cfd8f.png)

#### Expected behavior:

The border width should be consistent across the different input components.

### Priorities and help requested:

Are you willing to submit a PR to fix? Yes

Requested priority: Low

**Comment by kenotron** [comment]
@onlyann @betrue-final-final 
Would you two mind coordinating a PR here?

**Comment by onlyann** [comment]
@betrue-final-final , are all input components intended to have a `2px` border width when they receive focus with the normal theme? 
If so, updating [BasePicker styles](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/pickers/BasePicker.styles.ts) should do the trick AFAIK.

**Comment by onlyann** [comment]
`SearchBox` will also need an update.

**Comment by kenotron** [comment]
This seems to be intentional from that OP link: 

![](https://user-images.githubusercontent.com/20363037/64815800-42256d00-d55b-11e9-810e-57d0d9648a51.png)

**Comment by kenotron** [comment]
@betrue-final-final - can you take a look at this bug please?

**Comment by kenotron** [comment]
@betrue-final-final ping?

**Comment by micahgodbolt** [comment]
Ben did a pass on a bunch of controls recently increasing their focus state to 2 pixels. Looks like we still need to make this fix on 

- [ ] Pickers
- [ ] PeoplePicker
- [ ] DatePicker
- [ ] SearchBox

These are already done

- [ ] TextField
- [ ] Dropdown
- [ ] Combobox
- [ ] SpinButton
- [ ] ColorPicker
- [ ]

**Comment by xugao** [comment]
i will look into this, most likely not today but next week

**Comment by micahgodbolt** [comment]
Thanks Xu!
